supabase/migrations/098_mylist_status_notifications.sql

```
@@ -0,0 +1,90 @@
+        'follow_request',
+        'anime_recommendation',
+        'broadcast',
+        'mylist_status'
+    ));
+
+-- ----------------------------------------------------------------
+-- 未読取得高速化の部分インデックス（broadcast 用に倣う）
+-- ----------------------------------------------------------------
+CREATE INDEX IF NOT EXISTS notifications_unread_mylist_recipient_idx
+    ON public.notifications (recipient_id, created_at DESC)
+    WHERE type = 'mylist_status' AND NOT read;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)